apps/web/app/components/radio-card.jsx

```diff
@@ -18,91 +18,128 @@ const RadioCard = ({
   url,
   // eslint-disable-next-line react/prop-types
   country,
+  // eslint-disable-next-line react/prop-types
 }) => {
   const { player, setPlayer } = usePlayer();
+
   return (
+    //Main
     <div
-      key={stationuuid}
-      className={`bg-white rounded-lg shadow-md p-4 ${
-        player.stationId === stationuuid ? "animate-pulse" : ""
-      }`}
+      className={`flex flex-col flex-wrap max-w-sm mx-auto bg-white rounded-xl border border-gray-200 overflow-hidden p-4 h-[220px] w-[268px] flex-shrink-0 justify-between`}
```

Comment:
`width` ve `height` için pixel değeri verilmiş, alternatifi ne olabilir?

Reply:
Bunu çözeceğim ancak şu an çözemedim yarın i18'i çözdüğüm esnada buna da bakacağım